app/components/support_interface/application_choice_component.rb

```diff
@@ -97,6 +98,20 @@ def rejection_reason_row
       { key: 'Rejection reason', value: rejection_reasons_text }
     end
 
+    def offer_conditions_row
+      return unless application_choice.pending_conditions? || application_choice.offer?
+
+      conditions = application_choice.offer&.[]('conditions')
+      return unless conditions.present?
+
+      {
+        key: 'Conditions',
+        value: render(SupportInterface::ConditionsComponent.new(conditions: conditions)),
+        action: 'Change conditions',
+        action_path: support_interface_edit_application_choice_conditions_path(application_choice_id: @application_choice.id),
```

Comment:
Use ‘Change’ link with hidden text specifying ‘conditions’:

```suggestion
        action: 'conditions',
        change_path: support_interface_edit_application_choice_conditions_path(application_choice_id: @application_choice.id),
```